pkgs/racket-doc/scribblings/reference/chaperones.scrbl

```diff
@@ -142,7 +142,7 @@ chaperones to @racket[v2].
 For values that include no chaperones or other impersonators,
 @racket[v1] and @racket[v2] can be considered chaperones of each other
 if they are @racket[equal?], except that corresponding mutable
-vectors, boxes, strings, byte strings, @tech{mutable pairs}, and
+vectors, boxes, hashes, strings, byte strings, @tech{mutable pairs}, and
```

Comment:
Just noticed this small point: the reference terminology is "hash tables" instead of "hashes".